Libs/DICOM/Widgets/ctkDICOMBrowser.cpp

```diff
@@ -691,7 +704,9 @@ void ctkDICOMBrowser::setDatabaseDirectory(const QString& directory)
   }
 
   // pass DICOM database instance to Import widget
+#if QT_VERSION >= QT_VERSION_CHECK(5,0,0)
```

Comment:
The check should be done based on enabled/disabled features. We should avoid features disappearing silently based on detected Qt version.